### ISSUE TITLE
No Such File or Directory

### ISSUE THREAD

**Issue by zmanian** [dan-t/rusty-tags#27]
I entered a directory with a rust project. I'm in the same level as the `cargo.toml`.

Run `rusty-tags vi`

I get this error
```
Fetching source of dependencies ...
No such file or directory (os error 2)
```

I'm on Ubuntu 16.04 and rusty-tags v0.9.1

**Comment by dan-t** [comment]
> I entered a directory with a rust project. I'm in the same level as the cargo.toml.
> 
> Run rusty-tags vi
> 
> I get this error
> 
> Fetching source of dependencies ...
> No such file or directory (os error 2)

Are you using rusty-tags 0.9.1? If not, please test it with this version.

**Comment by zmanian** [comment]
Yes I'm 0.9.1

**Comment by dan-t** [comment]
Has your ctags version a `--recurse` option?

**Comment by zmanian** [comment]
Okay, I didn't realize there was a dependency on a ctags binary.

This could be resolved on ubuntu 16.04 by `sudo apt-get install exuberant-ctags`
